### PR TITLE
add unicode version

### PR DESCRIPTION
**Pull request by Carbon225**
- [x] emoji version
- [x] character version

Emoji version is different from "unicode version" afaik.
They will be reported in fields "emoji_version" and "unicode_version".

### CODE REVIEW

**Comment by Carbon225** [comment]
Also adds the unicode data downloader.

**Comment by Carbon225** [comment]
The grapheme unicode version is the minimum version that supports all its codepoints.

**Comment by Carbon225** [comment]
# Emoji version vs unicode version

Consider this emoji:
https://emojipedia.org/kiss-woman-man

It was added in Emoji 2.0, this is the emoji version.
What is its unicode version? I think it doesn't have one.

This character:
https://unicode.link/codepoint/1F91E-hand-with-index-and-middle-fingers-crossed

Unicode version 9.

https://emojipedia.org/crossed-fingers

But here it's both unicode 9.0 and emoji 3.0.

File: label_inspector/models.py [hunk 1]
```diff
@@ -63,6 +63,8 @@ class InspectorCharResult(BaseModel):
             '* `invisible` - zero width joiner or non-joiner (which is not part of correct emoji)\n'
             '* `special` - for any character that does not match one of the other classifications'
         )
+    unicode_version: Optional[str] = Field(description="Unicode Version of the character.\n"
+                                                        "* `null` if the character is not assigned to any version")
```
Comment (djstrong) [inline label_inspector/models.py:67]: could you add tests for null?

File: label_inspector/models.py [hunk 2]
```diff
@@ -100,6 +102,10 @@ class InspectorGraphemeResult(BaseModel):
 
     description: str = Field(description="Description of the grapheme type.")
 
+    unicode_version: Optional[str] = Field(description="Unicode Version of the grapheme.\n"
+                                                       "* `null` if the grapheme is not assigned to any version")
```
Comment (djstrong) [inline label_inspector/models.py:106]: could you add tests for null?

File: label_inspector/analysis/grapheme_analysis.py [hunk 3]
```diff
@@ -11,6 +11,10 @@
     from .label_analysis import LabelAnalysis
 
 
+def v2num(version: str) -> int:
+    return int(version.replace('.', ''))
```
Comment (djstrong) [inline label_inspector/analysis/grapheme_analysis.py]: there was a version 1.0.1 (not sure if reflected in files?). maybe better solution would be `(int(n) for n in version.split('.'))`?

**Review by djstrong — APPROVED** [review]
